@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sensor/fn_targetingFCRUpdate.sqf

```diff
@@ -73,7 +73,7 @@ private _fcrTargets = [];
     if ((_type != FCR_TYPE_FLYER && _type != FCR_TYPE_HELICOPTER) && _fcrMode == 2) then {continue;};
     if ((vectorMagnitude velocityModelSpace _target) < 5 && _fcrMode == 2) then {continue;};
 
-    _fcrTargets pushBack [getPosAsl _target, _type, speed _target, _target];
+    _fcrTargets pushBack [aimpos _target, _type, speed _target, _target];
```

Comment:
This doesn't seem related to the reticule?

Reply:
fixed